src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,56 @@
+# Navigate Between Application Windows
+
+Pop!\_OS uses the GNOME desktop environment, which includes keyboard shortcuts for quickly switching between system and application windows.
+
+---
+
+## Using the Dock
```

Comment:
There is a setting in Desktop -> Dock to show a menu with all of an apps' open windows (it's called `Icon Click Action`). Since that is a feature of the dock, and relates directly to switching between application windows, it seems like this might be an appropriate place to talk about it.

Reply:
This is outlined in the next section, "Customize Pop!_OS", but I can illustrate it here.